CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Unreleased (`master`)
+
+Drop official support for Ruby 2.3 and below, though these versions may continue
+to work.
+
+# 0.1.3
```

Comment:
Where did this changelog come from?

Reply:
I looked at the past commits and wrote one by hand.